templates/3c_generate_qc_metrics.ipynb

```diff
@@ -6,8 +6,15 @@
    "metadata": {},
    "source": [
     "# QC Notebook\n",
-    "## This notebook is an example: create a copy before running it or you will get merge conflicts!\n",
-    "This notebook can be run to compute and visualize the QC metrics for the run."
+    "## This notebook is an example: create a copy before running it or you will get merge conflicts!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf556c06-55eb-4ce1-a040-cb5260379244",
+   "metadata": {},
+   "source": [
+    "This notebook assesses the quality of the extracted imaged data by plotting intensity metrics per marker sorted by time of acquisition. This will help you identify anomalous changes that may indicate systematic error related to the instrument/extraction process, and not with the images themselves."
```

Comment:
```suggestion
    "This notebook assesses the quality of the extracted imaged data by plotting intensity metrics per marker sorted by time of acquisition. This will help you identify anomalous changes that may indicate systematic error related to the image acquisition process, and not with the samples themselves."
```

This makes it sound like we've found errors with the extraction process, which is not the case